addons/website_slides/models/slide_channel.py

```diff
@@ -601,3 +611,12 @@ def _resequence_slides(self, slide):
 
     def get_backend_menu_id(self):
         return self.env.ref('website_slides.website_slides_menu_root').id
+
+    def send_email_to_responsible(self):
+        mail_ids = []
+        for channel in self:
+            template = channel.enroll_request_template_id.with_context(
+                user=self.env.user
+            )
+            mail_ids.append(template.sudo().send_mail(channel.id))
```

Comment:
Why sending directly emails ? You could do a message_notify and let Odoo decide whether to use Inbox or email.